app/controllers/this_is_cool_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ThisIsCoolController < ApplicationController
+    before_action :authenticate_user!
+
```

Comment:
In Ruby, there is a language convention of using 2 space indentation. It looks like this file is using 4 space indentation, so you might want to check your editor config.